### ISSUE TITLE
[ADE Support] Screen reader issue: <select> items don't read on hover

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#103714]
> [!NOTE]
> **QUICK SUMMARY:** Desktop screen readers don't read items in `<select>` or `<va-select>` on hover. We should investigate this and try to fix it or propose alternative patterns to use so that those users are able to access all content in forms. See [Slack thread](https://dsva.slack.com/archives/C0335Q88X0S/p1740412990810699) for context.

## Issue Description
We encountered an issue where a user:
- used a screen reader with a mouse. The screen reader was set to mouse tracking, i.e., read items that the mouse is hovering on.
- clicked on a `<va-select>` component to access the items inside
- when hovering over the items, the screen reader **did not read them aloud**

I investigated this further by trying out different screen readers with mouse tracking turned on - VO, NVDA, and JAWS - with both `<va-select>` and vanilla HTML `<select>`. The results were the same each time - the screen reader doesn't read the items aloud on hover.

[Example from a product on staging, taken during a session with a live participant](https://github.com/user-attachments/assets/b35f7924-db31-4922-b926-dfed94a6413f)

[Example using va-select from the VADS Storybook, and select via the W3 Schools website](https://github.com/user-attachments/assets/0fc57f83-74d0-407b-883a-5901f632838c)

This is an issue for our user base - we likely have a higher proportion of screen reader + mouse users than the wider screen reader-using population because if if Vets lose their sight, it's as adults, and often gradually. And if we use `<va-select>` / `<select>` and that Vet can't access items within it, that can impede their ability to fill out forms independently and receive benefits.

## About the Team

- **Team name**: ADE
- **Product name:** n/a
- **Product manager:** Martha Wilkes
- **Slack Channel:** [accessibility-de-internal](https://dsva.slack.com/archives/C08B367JLLT)
- **Additional contact on this issue:** @BobbyBaileyRB (MHV)

### ADE Tasks
- [ ] Test `<select>` and `<va-select>` using mobile devices + screen readers + touch (not swipe). We need to determine if this hover issue is desktop-specific.
- [ ] Investigate possible fixes
- [ ] Propose alternate patterns if no fix is possible
- [ ] Let Bobby know where this lands.
- [ ] (what else?)

### Ticket Updates
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_ @artsymartha68 @lakper 
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`

### Related Tickets (ADE Internal)
> [!IMPORTANT]
> #### Tasks are Now Sub-Issues
> - Related tickets are no longer being tracked in task lists, due to the planned depreciation of tasks by GitHub in April 2025. 
> - Instead, related tickets will be connected to tickets via sub-issues.
> ##### Connecting Tickets
> - **To connect an existing ticket**, choose the drop-down from "create sub-issue" and select "add existing issue"
> - **To connect a new issue,** choose the drop-down from "create sub-issue" and select "create sub-issue," and select the respective template.

**Comment by lakper** [comment]
Issue logged via appropriate template, closing ticket.
